R/cancensus.R

```diff
@@ -12,10 +12,10 @@
 #' @param level The census aggregation level to retrieve, defaults to \code{"Regions"}. One of \code{"Regions"}, \code{"PR"}, \code{"CMA"}, \code{"CD"}, \code{"CSD"}, \code{"CT"}, \code{"DA"}, \code{"EA"} (for 1996), or \code{"DB"} (for 2001-2016).
 #' @param vectors An R vector containing the CensusMapper variable names of the census variables to download. If no vectors are specified only geographic data will get downloaded.
 #' @param geo_format By default is set to \code{NA} and appends no geographic information. To include geographic information with census data, specify one of either \code{"sf"} to return an \code{\link[sf]{sf}} object (requires the \code{sf} package) or \code{"sp"} to return a \code{\link[sp]{SpatialPolygonsDataFrame-class}} object (requires the \code{rgdal} package).
-#' @param resolution Resolution of the geographic data. {cancensus} will download simplified geometries by default. For lower level geometries like DB or DA this will be very close to the high resoltion data,
-#' siplification level increases as the geographic aggregation level increases.
+#' @param resolution Resolution of the geographic data. {cancensus} will download simplified geometries by default. For lower level geometries like DB or DA this will be very close to the high resolution data.
+#' Simplification generally increases as the geographic aggregation level increases.
 #' If high resolution geometries are required
-#' then this option can be set to 'high'. By default is set to \code{'simplified'}.
+#' then this option can be set to 'high'. By default this setting is set to \code{'simplified'}.
```

Comment:
Suggest "max"/"simplified" as an alternative to "high"/"simplified"

Reply:
I'd rather keep "high", all geographies have been processed. "max" implies that they have maximal resolution, but all geographies are simplified to some degree and they are also heavily processed in the sense that water features (simplified depending to geographic level) have been removed.

Reply:
sounds good, but I wonder if at some point we should be more transparent that the spatial data people are using is (1) not the StatCan master original and (2) simplified and therefore not suitable for anything other than simple thematic mapping and analysis. I am sure no body is making planning decisions using these geometries and maybe I'm just being a bit pedantic.